waiter/src/waiter/settings.clj

```diff
@@ -152,7 +153,10 @@
   (utils/map->json-response (into (sorted-map) (sanitize-settings settings))))
 
 (def settings-defaults
-  {:cors-config {:kind :patterns
+  {:authenticator-config {:kind :anonymous
+                          :anonymous {:factory-fn 'waiter.auth.authentication/anonymous-authenticator
+                                      :launch-as-user (System/getProperty "user.name")}}
```

Comment:
I would favor making `:launch-as-user` a required field when using `:anonymous` as opposed to defaulting it to the `user.name` property